platforms/chibios/drivers/audio_dac_basic.c

```diff
@@ -77,9 +86,9 @@ GPTConfig gpt7cfg1 = {.frequency = AUDIO_DAC_SAMPLE_RATE,
 
 static void gpt_audio_state_cb(GPTDriver *gptp);
 GPTConfig   gptStateUpdateCfg = {.frequency = 10,
-                               .callback  = gpt_audio_state_cb,
-                               .cr2       = TIM_CR2_MMS_1, /* MMS = 010 = TRGO on Update Event.    */
-                               .dier      = 0U};
+                                 .callback  = gpt_audio_state_cb,
+                                 .cr2       = TIM_CR2_MMS_1, /* MMS = 010 = TRGO on Update Event.    */
+                                 .dier      = 0U};
```

Comment:
Add a trailing comma here like this:
```suggestion
                                 .dier      = 0U,
};
```
Then clang-format will reformat this code in a way that's less dependent on a particular clang-format version.

Reply:
I've done this, and also made the same change in the lines above to keep the code consistent. I'm not sure if Vscode is formatting this correctly though

Reply:
If you can't run `qmk format-c` on the file, you can look at the diff generated by the lint job (e.g., https://github.com/qmk/qmk_firmware/actions/runs/7681682005/job/20935224989?pr=22974) and apply it manually (in this case apparently you need to use 4 spaces for the indent inside braces, where you have 6 at the moment).

Reply:
So interestingly running `qmk format-c` still gives me 6 spaces here.  It also seems to mess up lines 133-140, and lines 160-167. In all three cases adding a blank line between the static variable just above the code block resolves the issue. lines 133-140 and 160-167 revert and 92-96 moves to a 4 space indent. 

Apparently Ubuntu has an old version of clang-format. I think this might be the issue

Reply:
Okay so partial success, just checked with clang-format version 19.0.0 and i still get 6 spaces here but the other lines no longer change.

What version is the CI flow running with so i can match up to that?

Reply:
If you can run docker containers, prefix the format command with `./util/docker_cmd.sh` -- it'll use the same container as CI, clang-format included.

Reply:
Running through docker worked thats for the advice. I'll wait for the CI to confirm though